openwisp_users/models.py

```diff
@@ -59,6 +60,14 @@ def organizations_pk(self):
                     .values_list('organization_id')
         return qs
 
+    def clean(self):
+        if self.email:
```

Comment:
could you add a comment that recaps what I wrote in the issue description? I mean to write the reason for which we are doing application level validation and not using the DB constraint, it's explained in the description of #28